### ISSUE TITLE
Release 1.4.0

### ISSUE THREAD

**Issue by LeeAlephium** [alephium/desktop-wallet#378]
Modify this description to add or check off tasks needed to release 1.4.0.

Non-blockers should be their own issues.

[Testing spreadsheet](https://docs.google.com/spreadsheets/d/1sJ9-MJHhcx99auTENpJjWd7UdQtiTa0j8g9VtlOWGAc/edit?usp=sharing)

Issues to be resolved found from release testing:

* [x] Derive a changelog (at the very least to know what's new to test)
* [x] https://github.com/alephium/desktop-wallet/pull/379

I think this covers everything (I derived it from our commit history), let me know / edit the list:

* Internationalization support, with French as the first new language
* Accessibility support: the entire application can be navigated with a keyboard and screen reader
* Upgrade to use new network services (1.8 explorer and 1.4.5 Alephium node)
* A refresh of the underlying dependencies
* Redesign of how transactions are presented
  * Transactions between addresses of the same wallet are now condensed into one "Move" transaction
* Addresses over the entire application can be copied
* Lock times are now supported
  * Indicated with a lock beside the amount sent / received
  * When lock time passes the lock disappears
* The window theme syncs with the application theme
* Custom scrollbar to match across systems
* Misc. UI updates (address badges, ellipsed addresses, toggle sections)

**Comment by LeeAlephium** [comment]
@nop33 without fail I've forgotten once again how to test sweeps - could you describe how they're done and we can save it somewhere for the future and I can do it?

I noticed this when going in offline mode - everything turns to "zero":

**Comment by LeeAlephium** [comment]
You can delete an account but if it's selected it stays selected:


https://user-images.githubusercontent.com/105806603/192866988-f33a876a-2c43-40d6-9113-00718620bf44.mp4

**Comment by nop33** [comment]
> @nop33 without fail I've forgotten once again how to test sweeps - could you describe how they're done and we can save it somewhere for the future and I can do it?

I guess you mean how to test the consolidation feature through the send modal. You need to use the Swagger UI to build a couple of TX with many many outputs towards one of your addresses, so that when you try to send those funds back you'll get the consolidation modal poping up before sending the tx.

**Comment by LeeAlephium** [comment]
"Optional passphrase" text is too high / not enough spacing between the fields:

![1664395012](https://user-images.githubusercontent.com/105806603/192876625-dea6add5-028e-454e-becf-e62f5f9be03f.png)

**Comment by LeeAlephium** [comment]
An example of the incorrect amount calculation manifesting:

![1664395426](https://user-images.githubusercontent.com/105806603/192877881-ac8e453f-f6fb-42a0-8cbe-b5a06f93c575.png)

It says "0.001" locked but none is supposedly sent (I did send 0.001, not 0 + fee...)

**Comment by LeeAlephium** [comment]
On the wallet import screen, it requests the recovery phrase, but also says to write it down... This doesn't make sense since it'll already be written or stored somewhere since they are importing it.

![1664396948](https://user-images.githubusercontent.com/105806603/192882324-3b290915-1124-4ea4-a6ff-6f52d58175f7.png)

**Comment by LeeAlephium** [comment]
Accessibility issues:

* [ ] Pressing enter on passphrase toggle when wallet and password are entered submits the form instead of expanding the section
* [ ] Toggling the checkbox for "I understand that..." and then going to the passphrase input and typing doesnt work
* [ ] Copy buttons are being tabbed to in the transaction rows
* [ ] Double text being spoken (hidden text and the visible text) in the transaction rows
* [ ] Offline mode icon not tabbable (should be so blind user can listen to screen reader and know wallet is offline...)

**Comment by LeeAlephium** [comment]
@nop33 @mvaivre please add what you think is a blocker to the issue description and the rest (who knows, maybe all) can be created as separate issues

**Comment by LeeAlephium** [comment]
I can't seem to invoke the consolidate modal - I created a huge transaction twice and it wasn't enough apparently? And when sending funds within the wallet it hung for loooong time and eventually showed '???' for the balance:

![1664400960](https://user-images.githubusercontent.com/105806603/192893320-acf0bd0a-4490-437d-8878-81b7a230d999.png)

[This was the data](https://gist.github.com/LeeAlephium/5759cbe232ed753a50a521fec9caecd8) which I sent to `/wallets/:wallet-name/transfer`

**Comment by nop33** [comment]
Thanks a lot @LeeAlephium for your thorough tests!

This format of reporting issues is not very convenient for discussing them. It'd be useful if there were discussion threads like in PRs (I know, not possible in issues).

I propose we create issues for each one of the obvious ones. When unsure, just ping on Slack:

<img width="956" alt="image" src="https://user-images.githubusercontent.com/1579899/192978868-7eddc5d3-92a3-4b8b-9bdb-f9fa9b930d03.png">

**Comment by mvaivre** [comment]
Great testing session, thanks a lot @LeeAlephium! 🙏 
I'll create related issues.

**Comment by nop33** [comment]
Based on @LeeAlephium's changelog, I've updated it to add some more:

```
## 🎉 New features
- Transaction amounts can be locked until a specified date time #358
- Internationalization support, with French as the first new language #272
- Display pending sent transactions on app launch #323

## 🤝 Feedback from community
- Addresses can now be copied by hovering over their label #334 #302
- The application theme syncs with the OS theme #109

## ♿️ Accessibility
- The entire application can be navigated with a keyboard and screen reader #191

## 🎨 UI updates
- Improved display of transaction table rows #301
- Transactions between addresses of the same wallet are now condensed into one "Move" transaction #367
- Improved display of address labels #350
- Transaction details were extended to include lock time #358
- Custom scrollbar to match across systems #313
- Update expandable sections with a toggle button
- Update theme colors

## 🐛 Bug fixes
- Refresh list of available wallets once a wallet gets deleted #306

## Other updates
- Using new mainnet and testnet services #363
- Use consistent wording when mentioning the recovery phrase #315
- Updated dependencies
```
